### ISSUE TITLE
[Python] Implement new Concept Exercise: string splitting

### ISSUE THREAD

**Issue by DavidGerva** [exercism/v3#1648]
This issue describes how to implement the concept exercise, for the Python track, which should explain how to split strings in python.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This concept exercise should convey a basic understanding of the string's split methods in python.

## Learning objectives

- given a string, use the string's `split()` method to split it, also given a specific separator
- use the string's `split()` method with the maxsplit parameter 
- use the string's `splitlines()` method to split a string into its lines

## Out of scope

- Nothing

## Concepts

- string's `split()` method
- string's `splitlines()` method

## Prerequisites

- basic string concepts

## Resources to refer to

- [String official doc](https://docs.python.org/3.4/library/string.html)
- [Python string split()](https://www.programiz.com/python-programming/methods/string/split)
- [Python string splitlines() official doc](https://docs.python.org/3/library/stdtypes.html#str.splitlines)
- [Python string split() official doc](https://docs.python.org/3/library/stdtypes.html#str.split)

### Hints

- TBD?

### After

Nothing

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

Tests should be written using unittest.TestCase and the test file named bool_basic_test.py.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

## Edits
-

**Comment by BethanyG** [comment]
@DavidGerva @cmccandless  -- on second look, this feels like it should be part of `string methods`, and not its own exercise.  Thoughts?

**Comment by BethanyG** [comment]
Closing this issue in favor of either folding it into `string methods` or a `text processing` concept.
